hack/util.sh

```diff
@@ -609,9 +617,9 @@ function install_router {
                  --cert="${MASTER_CONFIG_DIR}/router.crt" --key="${MASTER_CONFIG_DIR}/router.key"
             cat "${MASTER_CONFIG_DIR}/router.crt" "${MASTER_CONFIG_DIR}/router.key" \
                 "${MASTER_CONFIG_DIR}/ca.crt" > "${MASTER_CONFIG_DIR}/router.pem"
-            ROUTER_DEFAULT_CERT="--default-cert=${MASTER_CONFIG_DIR}/router.pem"
+            router_args+=("--default-cert='${MASTER_CONFIG_DIR}/router.pem'")
```

Comment:
You're overquoting.  This adds an element with literal quotation marks, and when you use `${router_args[*]}` below, you'll pass the quotation marks to `openshift admin router`.  See the following example:

```console
$ foo=( --foo --bar ); foo+=( "--baz='quux'" ); echo args: ${foo[*]}
args: --foo --bar --baz='quux'
```

Does `openshift admin router` strip quotation marks from the `--default-cert` flag value?